components/textfield/metadata/textarea.yml

```diff
@@ -8,37 +8,124 @@ sections:
      See the [Textfield migration guide](textfield.html#migrationguide) for migration information.
 examples:
   - id: textfield
-    name: Standard
+    name: Standard Sizes
+    markup: |
+      <div class="spectrum-Textfield spectrum-Textfield--sizeS spectrum-Textfield--multiline">
+        <label for="textfield-sm" class="spectrum-FieldLabel spectrum-FieldLabel--sizeS">Password</label>
```

Comment:
Password seems like a somewhat awkward example for a textarea as it kind of makes the height of the box seem like an accident. Maybe an example like `Feedback`?